wiki/entry_hahn.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Wiki Entry Template"
+ subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+ author: "Alina Hahn"
+ categories:
+- "Digital Media"
+---
+
```

Comment:
Please try to follow the KARQ and summary structure as requested in the exercise!

Reply:
I was confused about the structure because normally I have to write a whole essay and not single sections. Also, unfortunately, I wasn't in the class when the exercise was discussed and the exercise wasn't 100% clear to me.  Although I didn't write the KARQs in individual sections, I followed the order (first KARQ, second summary) in my paper. Thank you for your comment, I will keep it in mind for future KARQs.

Reply:
Research Area: Digital media use

Text: Meier, A., & Reinecke, L. (2021). Computer-mediated communication, social media, and 	mental health: A conceptual and empirical meta-review. Communication 				Research, 48(8), 1182-1209.

Reply:
_How do the effects of CMC and social media on mental health vary across different populations (e.g. gender, age, cultural or social background), and how can future studies research these differences?’._

Future research should definitely try to find links between effects that CMC and social media can have on mental health. When studying the effects on different populations it could look at personal wellbeing (PWB) and on  psychopathology (PTH) and thus try to work out wether there are also differences in age groups or gender etc.